service/src/main/java/com/epam/rft/atsy/service/impl/UserServiceImpl.java

```diff
@@ -54,8 +56,13 @@ public Long saveOrUpdate(UserDTO userDTO) {
         UserEntity entity = modelMapper.map(userDTO, UserEntity.class);
         try {
             return userRepository.save(entity).getId();
-        } catch (ConstraintViolationException | DataIntegrityViolationException constraint) {
-            throw new DuplicateRecordException("User already exists!");
+        } catch (ConstraintViolationException | DataIntegrityViolationException ex) {
+            log.error("Save to repository failed.", ex);
```

Comment:
There is already a LOGGER static final attribute in the class, by introducing @Slf4j that attribute gets deprecated.
Please remove it and rewrite its usage to the new log attribute.